tests/validation/rules/slackChannelName.spec.ts

```diff
@@ -47,7 +47,7 @@ describe('validation/rules/slackChannelName', () => {
     expect(rule.$validator({}, null, null)).toBe(false)
   })
 
-  test('it validates whether the value is valid with offset', () => {
+  it.skip('validates the value with an offset', () => {
```

Comment:
Seems the few test cases here are also skipped for no reason?

Reply:
seems like I skipped them while developing but forgot to remove it later